pkg/actuators/machine/actuator.go

```diff
@@ -233,8 +240,55 @@ func (a *Actuator) Delete(context context.Context, cluster *clusterv1.Cluster, m
 	return nil
 }
 
+type glogLogger struct{}
+
+func (gl *glogLogger) Log(v ...interface{}) {
+	glog.Info(v...)
+}
+
+func (gl *glogLogger) Logf(format string, v ...interface{}) {
+	glog.Infof(format, v...)
```

Comment:
Doesn't save you much work, but you can import [`go-log/log/info`][1] instead of creating this local `glogLogger`.

[1]: https://godoc.org/github.com/go-log/log/info

Reply:
The goal was to use only a single logger wherever possible. I did not want to introduce another one. Thus, creating a special (not super cool at all) data type just for this.

Reply:
> The goal was to use only a single logger...

So you're avoiding `info.New(glog.V(0))`?

[1]: https://github.com/go-log/log/blob/master/info/info.go#L24

Reply:
Did not know you can do it this way. On it.